modules/internal/ChapelRange.chpl

```diff
@@ -55,7 +60,17 @@ module ChapelRange {
     the flow of the document better.
    */
   pragma "no doc"
-  enum BoundedRangeType { bounded, boundedLow, boundedHigh, boundedNone };
+  @deprecated("BoundedRangeType is deprecated; please use 'boundKind' instead")
+  proc BoundedRangeType type do return boundKind;
+//was: enum BoundedRangeType { bounded, boundedLow, boundedHigh, boundedNone };
+  @deprecated("'bounded' is deprecated; please use 'boundKind.both' instead")
+  proc type boundKind.bounded param do return boundKind.both;
```

Comment:
Oh very cool, I didn't realize you could add this kind of "alias" of one enum value to another.